pages/virtual-leagues/[type].tsx

```diff
@@ -1,32 +1,45 @@
 import { NextPage, NextPageContext } from "next";
 import VirtualLeaguePageContent from "../../components/virtual-leagues/VirtualLeaguePageContent";
-import {
-  getVerifiedEntries as getVirtualLeagues,
-  GetVerifiedEntriesResponse,
-} from "../../services/verified";
+import { getVerifiedEntries, GetVerifiedEntriesResponse, } from "../../services/verified";
 import { toVerifiedType, VerifiedType } from "../../services/VerifiedType";
 import { getVerifiedExtraInformation } from "../../utils/verifiedTypeHelper";
 
-interface PageProps {
+type LegalVerifiedType = {
   res: GetVerifiedEntriesResponse;
-  verifiedType: VerifiedType;
-}
+  verifiedType: VerifiedType
+  type: "LEGALVERIFIEDTYPE"
+};
+
+type IllegalVerifiedType = {
+  illegalVerifiedType: string,
+  type: "ILLEGALVERIFIEDTYPE"
+};
 
-const VirtualLeaguePage: NextPage<PageProps> = ({ res, verifiedType }) => {
-  if (res.type === "SUCCESS") {
+type VirtualLeaguePageProps = LegalVerifiedType | IllegalVerifiedType;
```

Comment:
Tror det blir penere hvis du joiner den union-typen her med hoved-typen til res, i stedet for å ha en nøsta type. Dvs
```
export type GetVerifiedEntriesResponse =
  | InvalidVerifiedType
  | GetVerifiedEntriesSuccess
  | GetVerifiedEntriesError;
```
Samme som i pages/verified/pl/slug

Reply:
Hvordan får jeg dekket behovet for disse da? Forsvinner ikke de da?

```js
  verifiedType: VerifiedType
  type: "LEGALVERIFIEDTYPE"
```

Reply:
jeg er dårlig å forklare, men sånn mente jeg https://github.com/fplbot/frontend/pull/41

Reply:
Åja, flate ut strukturen istedet for å sende med hele responsen! Jepp, fikser. Enig at det er penere.

Reply:
180c803